b2sdk/_test_manager/__init__.py

```diff
@@ -0,0 +1,9 @@
+######################################################################
```

Comment:
so it also seems to me that creating a separate package would be a cleaner approach, something like `b2sdk-test`?

Reply:
After a moments's thought, I concluded that we could actually go down the most common path I've encountered, which is simply package the test in the main package, so it would be b2sdk/test. Numpy, pandas or django use such a patter. What do you think?